src/styles/App.css

```diff
@@ -0,0 +1,51 @@
+@import url("https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Inter:wght@300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Montserrat:wght@200;300;400;500;600;700;800&family=Roboto+Slab:wght@100&family=Roboto:wght@700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  outline: none;
+}
+
```

Comment:
- [ ] I will suggest kindly that you remove all the redundant code from this file in order to make your codebase profesional and concise; you repeated all the code here in index.css file. ℹ️ This is against DRY principle. 👍